### ISSUE TITLE
mothur and other imports in read_phyloseq

### ISSUE THREAD

**Issue by antagomir** [microbiome/microbiome#38]
According to some feedback we have received, the phyloseq package does not provide full capabilities for mothur (and qiime?) imports, and updates to the package may be slow. We were asked to expand the read_phyloseq function so that it can also import simple Mothur OTU + Taxonomy tables in addition to simple CSV:s. That format is also different from the standard Mothur import of the phyloseq package and would hence provide a useful complementary tool. Also other simple table formats could be added as an option.

**Comment by microsud** [comment]
This is also important. I have not worked with simple mothur files. I will try to find more details.

**Comment by antagomir** [comment]
Yes the read_phyloseq from simple CSV would be primary but could be easy to expand to some other standard forms.

**Comment by microsud** [comment]
@antagomir I have added a simple function for immediate use using mothur files.

**Comment by antagomir** [comment]
Perfect, thanks! I added some improvements. Now it remains to add support to read simple CSVs generated by write_phyloseq, so that those files can be read back as well. This should go in the function read_phyloseq function when time allows.

**Comment by microsud** [comment]
Added support for converting different formats of OTU tables obtained from Mothur (shared and consensus taxonomy files), QIIME (biom and mapping files), and simple ".csv" format (otu_table, mapping and taxonomy files) into a phyloseq object. This is the simplified way to convert different data sets into a phyloseq object. Later we can also include ability to merge phylogenetic tree. However, at the moment this is easily done in phyloseq. For now I think we can close this issue?

**Comment by antagomir** [comment]
Excellent. This was fast and will be very useful. 

I added an example to vignettes/Data.Rmd (let us keep explalnation of all exported functions in the tutorial); and removed "print" commands from the functions. 

I also changed the "simple" format as default in read_phyloseq since this could be expected as write_phyloseq and read_phyloseq form a pair. 

Let us keep this open until ./inst/extras/build.sh build/check does not show errors (I am sending email about details).
